Mathlib/Algebra/Group/Center.lean

```diff
@@ -173,12 +173,12 @@ theorem zero_mem_center [MulZeroClass M] : (0 : M) ∈ Set.center M where
 variable {M}
 
 @[to_additive (attr := simp) neg_mem_addCenter]
-theorem inv_mem_center [Group M] {a : M} (ha : a ∈ Set.center M) : a⁻¹ ∈ Set.center M := by
+theorem inv_mem_center [DivisionMonoid M] {a : M} (ha : a ∈ Set.center M) : a⁻¹ ∈ Set.center M := by
   rw [_root_.Semigroup.mem_center_iff]
   intro _
   rw [← inv_inj, mul_inv_rev, inv_inv, ha.comm, mul_inv_rev, inv_inv]
-#align set.inv_mem_center Set.inv_mem_center
-#align set.neg_mem_add_center Set.neg_mem_addCenter
+#align set.inv_mem_center Set.inv_mem_centerₓ
+#align set.neg_mem_add_center Set.neg_mem_addCenterₓ
```

Comment:
Really not sure what the point of this is. You're just teaching people porting Lean 3 code that `ₓ` is noise that is basically never relevant